app/api/apply/route.ts

```diff
@@ -0,0 +1,100 @@
+import { NextRequest, NextResponse } from "next/server"
+import { google } from "googleapis"
+
+export const runtime = "nodejs"
+export const dynamic = "force-dynamic"
+export const revalidate = 0
+
+export async function POST(req: NextRequest) {
+  try {
+    const body = await req.json()
```

Comment:
Better to add the ApplicationData type, it makes things safer and keeps the fields consistent.

```
ex:-

type ApplicationData = {
  email: string
  fullName: string
  studentId: string
  academicYear: string
  semester: string
  specialization: string
  whatsapp: string
  linkedin: string
  github: string
  reason: string
  otherClubs?: string
  preferredTeam?: string
}
```

```suggestion
    const body: ApplicationData = await req.json()
```